daemon/lib/swaps/client.go

```diff
@@ -0,0 +1,30 @@
+package swaps
+
+import (
+	"time"
+
+	"github.com/40acres/40swap/daemon/database/models"
+	"github.com/shopspring/decimal"
+)
+
+type SwapClient interface {
+	CreateSwapOut(address string, amount string) error
```

Comment:
Hmm, worth researching for future to have a Money type, we will have:
- Liquid (SATS can be used)
- BTC
- LN BTC
- Taproot Assets (amount is in fact, ints of the asset)

Reply:
Its coming xD

Reply:
https://github.com/40acres/40swap/pull/35